SpatialGDK/Source/SpatialGDK/Private/LoadBalancing/LayeredLBStrategy.cpp

```diff
@@ -40,7 +40,17 @@ void ULayeredLBStrategy::Init()
 		const FName& LayerName = Layer.Key;
 		const FLayerInfo& LayerInfo = Layer.Value;
 
-		UAbstractLBStrategy* LBStrategy = NewObject<UAbstractLBStrategy>(this, LayerInfo.LoadBalanceStrategy);
+		UAbstractLBStrategy* LBStrategy;
+		if (LayerInfo.LoadBalanceStrategy == nullptr)
+		{
+			UE_LOG(LogLayeredLBStrategy, Error, TEXT("WorkerLayer %s does not specify a loadbalancing strategy (or it cannot be resolved). Using a 1x1 grid."), *LayerName.ToString());
+			LBStrategy = NewObject<UGridBasedLBStrategy>(this);
+		}
+		else
+		{
+			LBStrategy = NewObject<UAbstractLBStrategy>(this, LayerInfo.LoadBalanceStrategy);
+		}
```

Comment:
I'd rather we prevented PIE if this is unset, but this seems like a good backup

Reply:
Yeah, not sure if we have a good mechanism for that.
Also this is the same thing we do for the default layer loadbalancing strategy.